build.gradle

```diff
@@ -42,12 +42,13 @@ artifactory {
 
 jenkinsPlugin {
     pluginId = 'synopsys-security-scan-plugin'
-    jenkinsVersion = '2.377'
+    jenkinsVersion = '2.401.3'
```

Comment:
@kishorikumar This version will still have spring-security-web high vuln. A patch version of 5.8.5 needs to be added. cc: @devmehtasynopsys